src/org/typo3/chefci/helpers/Slack.groovy

```diff
@@ -0,0 +1,42 @@
+package org.typo3.chefci.helpers
+
+class Slack implements Serializable {
+    def script
+
+    JenkinsHelper jenkinsHelper
+
+    Slack(script) {
+        this.script = script
+        jenkinsHelper = new JenkinsHelper(script)
```

Comment:
maybe not important here, but a good rule of thumb: never use `new` in a constructor - rather inject it as a constructor parameter (or inject it with a setter if optional). This makes your class a lot better testable... but ok - here it is not that important...